admin_manual/configuration_server/security_setup_warnings.rst

```diff
@@ -0,0 +1,48 @@
+======================
+Warnings on Admin Page
+======================
+
+Your ownCloud server has a built-in configuration checker, and it reports its 
+findings at the top of your Admin page. These are some of the warnings you 
+might see, and what to do about them.
+
+.. figure:: ../images/security-setup-warning-1.png
+
+Cache Warnings
+--------------
+
+"No memory cache has been configured. To enhance your performance please 
+configure a memcache if available." PHP has three caching extentions: APCu, 
+OPCache, and Xcache. You will see this warning if you have no caches installed, 
+and you may see it if you have APCu < 4.0.6 installed; older versions are 
+disabled because of performance problems. 
```

Comment:
https://github.com/owncloud/core/commit/b701bbd8c5f6051ec0f4bd360473290cf9ebdf71 ...doesn't seem so. So we only block those versions and the user can install a newer version on their own.